src/precice/tests/ParallelTests.cpp

```diff
@@ -176,6 +176,105 @@ BOOST_AUTO_TEST_CASE(ConstructAndExplicitFinalize)
 
 BOOST_AUTO_TEST_SUITE_END()
 
+// In order to test enforced gather scatter communication with an empty master rank (see below)
+void runTestEnforceGatherScatter(std::vector<double> masterPartition, std::string configFile)
+{
+  PRECICE_TEST("ParallelSolver"_on(2_ranks), "SerialSolver"_on(1_rank));
+  std::string configFilename = configFile;
+
+  if (context.isNamed("ParallelSolver")) {
+    // Get mesh and data IDs
+    SolverInterface interface(context.name, configFilename, context.rank, context.size);
+    const int       meshID      = interface.getMeshID("ParallelMesh");
+    const int       writeDataID = interface.getDataID("MyData1", meshID);
+    const int       readDataID  = interface.getDataID("MyData2", meshID);
+    const int       dim         = interface.getDimensions();
+    BOOST_TEST(dim == 2);
+
+    // Set coordinates, master according to input argument
+    const std::vector<double> coordinates = context.isMaster() ? masterPartition : std::vector<double>({0.0, 0.5, 0.0, 3.5, 0.0, 5.0});
+    const unsigned int        size        = coordinates.size() / dim;
+    std::vector<int>          ids(size, 0);
+
+    // Set mesh vertices
+    interface.setMeshVertices(meshID, size, coordinates.data(), ids.data());
+
+    // Initialize the solverinterface
+    double dt = interface.initialize();
+
+    // Create some dummy writeData
+    std::vector<double> writeData;
+    for (unsigned int i = 0; i < size; ++i)
+      writeData.emplace_back(i + 1);
```

Comment:
We decided to always use `{` blocks `}` after after control structures.
There already have been too many painful errors because of that style.
Could you please apply that to the whole test?

Reply:
Done